lib/main.dart

```diff
@@ -238,6 +241,24 @@ class _ThaliAppState extends State<ThaliApp> {
                       )..load(),
                       lazy: false,
                     ),
+                    BlocProvider(
+                      create: (_) => CommitteesCubit(
+                        authState.apiRepository,
+                      )..load(),
+                      lazy: false,
+                    ),
+                    BlocProvider(
+                      create: (_) => BoardsCubit(
+                        authState.apiRepository,
+                      )..load(),
+                      lazy: false,
+                    ),
```

Comment:
Perhaps boards and societies can be lazy here, since they won't be displayed immediately and not very often.